src/CMakeLists.txt

```diff
@@ -1,45 +1,22 @@
-option(CPP_STARTER_USE_QT "Enable compilation of QT sample" OFF)
-option(CPP_STARTER_USE_FLTK "Enable compilation of FLTK sample" OFF)
-option(CPP_STARTER_USE_GTKMM "Enable compilation of GTKMM sample" OFF)
-option(CPP_STARTER_USE_IMGUI "Enable compilation of ImGui sample" OFF)
-option(CPP_STARTER_USE_NANA "Enable compilation of Nana GUI sample" OFF)
-option(CPP_STARTER_USE_SDL "Enable compilation of SDL sample" OFF)
+# uncomment to enable the gui examples
 
 # sdl
-if(CPP_STARTER_USE_SDL)
-  message("Using SDL2")
-  add_subdirectory(sdl)
-endif()
+# add_subdirectory(sdl)
 
 # qt
-if(CPP_STARTER_USE_QT)
-  message("Using Qt")
-  add_subdirectory(qt)
-endif()
+# add_subdirectory(qt)
 
 # fltk test
-if(CPP_STARTER_USE_FLTK)
-  message("Using FLTK")
-  add_subdirectory(fltk)
-endif()
+# add_subdirectory(fltk)
 
 # gtkmm test
-if(CPP_STARTER_USE_GTKMM)
-  message("Using gtkmm")
-  add_subdirectory(gtkmm)
-endif()
+# add_subdirectory(gtkmm)
 
 # imgui example
-if(CPP_STARTER_USE_IMGUI)
-  message("Using Dear ImGui")
-  add_subdirectory(imgui)
-endif()
+# add_subdirectory(imgui)
 
 # Nana
-if(CPP_STARTER_USE_NANA)
-  message("Using Nana")
-  add_subdirectory(nana)
-endif()
+# add_subdirectory(nana)
```

Comment:
Since the GUI examples are moving into their own repos, maybe all the comments from lines 0-19 should be removed? Should we delete the `src/nana`, `imgui`, `gtkmm` etc. directories as well?

Reply:
Yes, eventually will do.